rabbitmq_integration_test.go

```diff
@@ -318,51 +312,81 @@ func TestRabbitMqClient_PublishWithContext(t *testing.T) {
 }
 
 func TestRabbitMqClient_Close(t *testing.T) {
-	t.Run("ShouldRetunNoError_WhenIsClosedTrue", func(t *testing.T) {
+	t.Run("ShouldRetunNoError_WhenCloseRabbitMq", func(t *testing.T) {
```

Comment:
@bintangsibarani better rename the test to: `ShouldReturnNoError_WhenCloseCalledTwice` wdyt?